src/vos/vos_layout.h

```diff
@@ -295,8 +295,12 @@ struct vos_krec_df {
 	/** Incarnation log for key */
 	struct ilog_df			kr_ilog;
 	union {
-		/** btree root under the key */
-		struct btr_root			kr_btr;
+		struct {
+			/** btree root under the key */
+			struct btr_root			kr_btr;
+			/** Offset of known existing akey */
+			umem_off_t			kr_known_akey;
```

Comment:
Can we add some compile check when refresh the patch or in subsequent patch? such as:
D_CASSERT(sizeof(struct evt_root) >= sizeof(struct btr_root) + sizeof(umem_off_t));

Reply:
Yeah, I can do that.